docs/architecture/cloud-native/host-eshoponcontainers-application.md

```diff
@@ -1,27 +1,42 @@
 ---
 title: Hosting the eShopOnContainers application
-description: Architecting Cloud Native .NET Apps for Azure | Hosting the eShopOnContainers Application
+description: Architecting Cloud-native .NET Apps for Azure | Hosting the eShopOnContainers Application
 ms.date: 06/30/2019
 ---
 # Hosting the eShopOnContainers application
 
-The logic supporting the eShopOnContainers application can be supported by Azure using a variety of services. The recommended approach is to leverage Kubernetes using Azure Kubernetes Service. This can be combined with Helm deployment to ensure easily repeated infrastructure configuration. Optionally, developers can leverage DevSpaces for Kubernetes as part of their development process. Another option is to host the functionality of the app using Azure Serverless features like Azure Functions and Azure Logic Apps.
+The logic supporting the eShopOnContainers application can be supported by Azure using a variety of services. The recommended approach is to leverage Kubernetes using Azure Kubernetes Service (AKS). This can be combined with Helm deployment to ensure easily repeated infrastructure configuration. Optionally, developers can leverage DevSpaces for Kubernetes as part of their development process. Another option is to host the functionality of the app using Azure Serverless features like Azure Functions and Azure Logic Apps.
 
 ## Azure Kubernetes Service
 
-asdf
+If you'd like to host the eShopOnContainers application in your own AKS cluster, the first step is to create your cluster. You can do this using the Azure Portal, which will walk you through the required steps, or you can use the Azure CLI, taking care to ensure you enable Role-Based Access Control (RBAC) and application routing when you do so. The eShopOnContainers' documentation describes the steps involved in creating your own AKS cluster. Once the cluster is created, you must enable access to the Kubernetes dashboard, at which point you should be able to browse to the Kubernetes dashboard to manage the cluster.
+
+Once the cluster has been created and configured, you can deploy the application to it using Helm and Tiller.
```

Comment:
You can link to this WIKI: https://github.com/dotnet-architecture/eShopOnContainers/wiki/Deploy-to-Azure-Kubernetes-Service-(AKS) for deploying eShop on AKS